auth-web/src/components/auth/CreateAccount/AccountCreateBasic.vue

```diff
@@ -105,55 +118,36 @@ export default class AccountCreateBasic extends Mixins(Steppable) {
   private async save () {
     // Validate form, and then create an team with this user a member
     if (this.isFormValid()) {
-      const org:Organization = { name: this.orgName, orgType: Account.BASIC }
-      this.setCurrentOrganization(org)
-      if (!this.stepForward) {
+      const avaialble = await this.isOrgNameAvailable(this.orgName)
+      if (!avaialble) {
+        this.errorMessage =
+                'An account with this name already exists. Try a different account name.'
+        return
+      }
+      if (this.isAccountChange) {
         try {
+          const org: Organization = { name: this.orgName, orgType: Account.BASIC, id: this.currentOrganization.id }
+          this.setCurrentOrganization(org)
           this.saving = true
-          const organization = await this.createOrg()
+          const organization = await this.changeOrgType('downgrade')
           await this.syncOrganization(organization.id)
-          await this.syncMembership(organization.id)
+          // await this.syncMembership(organization.id)
           this.$store.commit('updateHeader')
-          this.redirectToNext(organization)
+          this.$router.push('/setup-account-success')
+          return
         } catch (err) {
           this.saving = false
-          switch (err.response.status) {
-            case 409:
-              this.errorMessage =
-                      'An account with this name already exists. Try a different account name.'
-              break
-            case 400:
-              if (err.response.data.code === 'MAX_NUMBER_OF_ORGS_LIMIT') {
-                this.errorMessage = 'Maximum number of accounts reached'
-              } else {
-                this.errorMessage = 'Invalid account name'
-              }
-              break
-            default:
-              this.errorMessage =
-                      'An error occurred while attempting to create your account.'
-          }
+          this.errorMessage = 'An error occurred while attempting to create your account.'
```

Comment:
Error messaging can be better I think, as from user's perspective they are not 'creating' account.